bootstrap.php

```diff
@@ -22,6 +22,11 @@ function () {
 							define( 'NFD_HELPCENTER_DIR', __DIR__ );
 							define( 'NFD_HELPCENTER_BUILD_DIR', __DIR__ . '/build/' );
 							define( 'NFD_HELPCENTER_PLUGIN_URL', $container->plugin()->url );
+							// Check if the 'enable_helpcenter' query parameter is present in the URL
```

Comment:
Should this feature not come from capabilities instead of a flag in bootstrap ?

Reply:
This is a temporary check to enable the helpcenter on onboarding module steps.